fastai/basic_data.py

```diff
@@ -269,10 +269,12 @@ def sanity_check(self):
             warn(message)
             print(final_message)
 
-def load_data(path:PathOrStr, fname:str='data_save.pkl', bs:int=64, val_bs:int=None, num_workers:int=defaults.cpus,
-                  dl_tfms:Optional[Collection[Callable]]=None, device:torch.device=None, collate_fn:Callable=data_collate,
-                  no_check:bool=False, **kwargs)->DataBunch:
-    "Load from `path/fname` a saved `DataBunch`."
-    ll = torch.load(Path(path)/fname, map_location='cpu') if defaults.device == torch.device('cpu') else torch.load(Path(path)/fname)
+def load_data(path:PathOrStr, file:PathLikeOrBinaryStream= 'data_save.pkl', bs:int=64, val_bs:int=None, num_workers:int=defaults.cpus,
+              dl_tfms:Optional[Collection[Callable]]=None, device:torch.device=None, collate_fn:Callable=data_collate,
+              no_check:bool=False, **kwargs)->DataBunch:
+    """Load from `path/file` a saved `DataBunch`.
```

Comment:
fastai style is only one shortish line "about" docstring with the rest of the details inside the doc notebooks.